idd/Energy+.idd.in

```diff
@@ -72716,11 +72716,14 @@ Boiler:HotWater,
        \type real
        \minimum> 0.0
        \default 1.0
-  A8 ; \field End-Use Subcategory
+  A8 , \field End-Use Subcategory
        \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
        \type alpha
        \retaincase
        \default General
+  N10; \field Off Cycle Parasitic Fuel Load
+       \units W
+       \note parasitic fuel load when the boiler is not operating (i.e., standing pilot)
```

Comment:
There is a new field here in Boiler:HotWater. I did not see this field added to any example file (or unit test) and I think it would be a good idea to add this field to one of the examle files and add the new output reports to track any future changes.

Reply:
Agreed, good catch @rraustad